cablelabs/snaps-boot
# Support for SC

What does this PR do?
This PR has the fixes for supporting Static clean. It will rollback the static changes and make IP DHCP from static for the given interface even if Management interface is not given.

Do you have any concerns with this PR?
No

How can the reviewer verify this PR?
A reviewer can verify this issue by giving Management interface NULL.
First, he needs to execute -s option to make IP static.
afterward, he has to execute -sc option to rollback the static changes and make IP static.

Any background context you want to provide?
NA

Screenshots or logs (if appropriate)
NA

Questions:
Have you connected this PR to the issue it resolves?
NA
Does the documentation need an update?
No
Does this add new Python dependencies?
No
Have you added unit or functional tests for this PR?
snap-boot execution with -sc option.
